Guidelines.md

```diff
@@ -365,7 +365,7 @@ Services SHOULD return the following response headers, except where noted in the
 
 Response Header    | Required                                      | Description
 ------------------ | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-Date               | All responses                                 | The date the request was processed, in [RFC 3339][rfc-3339] format
+Date               | All responses                                 | Timestamp the response was processed, based on the server's clock, in [RFC 5322][rfc-5322-3-3] date and time format.  This is mandatory for response headers.  Coordinated Universal Time is the preferred time zone reference for this header,  designated as either `GMT` or `+0000`.  For example: `Wed, 24 Aug 2016 18:41:30 GMT`
 Content-Type       | All responses                                 | The content type
```

Comment:
Actually, just found that [RFC2616§3.3.1](https://tools.ietf.org/html/rfc2616) says:

> All HTTP date/time stamps MUST be represented in Greenwich Mean Time
   (GMT), without exception. For the purposes of HTTP, GMT is exactly
   equal to UTC (Coordinated Universal Time). This is indicated in the
   first two formats by the inclusion of "GMT" as the three-letter
   abbreviation for time zone, and MUST be assumed when reading the
   asctime format.

So, I will update accordingly.